src/saml.ts

```diff
@@ -234,7 +235,7 @@ class SAML {
     const id = this.options.generateUniqueId();
     const instant = generateInstant();
 
-    if (this.options.validateInResponseTo) {
+    if (![false, ValidateInResponseTo.never].includes(this.options.validateInResponseTo)) {
```

Comment:
I appreciate this might be easier to code, but negations are typically easy to miss when working with the code later and require more cognitive load. Let's keep the same pattern and leave it as an affirmative.

Side note: I'm glad to be getting rid of opportunities for truthy and falsey to be valid answers. I'd much rather see `=== true` or `=== false` for a security product, so that we are 100% clear what we are processing and what the consumer is sending (no chance of `"false"` being true).

Reply:
Another option is do to as @markstos suggested [here](https://github.com/node-saml/node-saml/pull/37#pullrequestreview-887974520) and convert this to a function, perhaps `willValidateInResponseTo()`. In that case, negation wouldn't be as likely to get lost in all the syntax (though I'd still prefer to avoid negation where possible).

Reply:
I was working on these changes while you commented. I created this function as you can see in the new version.